csm_web/frontend/src/components/App.js

```diff
@@ -1,62 +1,20 @@
 import React from "react";
 import { MemoryRouter as Router, Route, Switch, Link } from "react-router-dom";
 import ReactDOM from "react-dom";
-import Section from "./Section";
 import CourseMenu from "./CourseMenu";
-import { fetchJSON } from "../utils/api";
+import Home from "./Home";
 import LogoNoText from "../../static/frontend/img/logo_no_text.svg";
 
 export default class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { studentProfiles: null, mentorProfiles: null, currentProfile: null, ready: false };
-    this.fetchProfiles = this.fetchProfiles.bind(this);
-  }
-
-  fetchProfiles() {
-    fetchJSON("profiles/").then(({ studentProfiles, mentorProfiles }) => {
-      this.setState({
-        studentProfiles,
-        mentorProfiles,
-        currentProfile: studentProfiles[0] || mentorProfiles[0],
-        ready: true
-      });
-    });
-  }
-
-  componentDidMount() {
-    this.fetchProfiles();
-  }
-
   render() {
-    if (!this.state.ready) {
-      return <div>Loading...</div>;
-    }
-    const { currentProfile, mentorProfiles, studentProfiles } = this.state;
     return (
       <Router>
         <React.Fragment>
           <Header />
           <main>
             <Switch>
               <Route exact path="/" component={Home} />
-              <Route
-                path="/sections/:id"
-                render={routeProps => (
-                  <Section
-                    key={routeProps.match.params.id}
-                    currentProfileId={
-                      mentorProfiles
-                        .concat(studentProfiles)
-                        .filter(profile => profile.section == routeProps.match.params.id)[0].id
-                    }
-                    isMentor={mentorProfiles
-                      .map(profile => profile.section)
-                      .includes(Number(routeProps.match.params.id))}
-                    {...routeProps}
-                  />
-                )}
-              />
+              <Route path="/sections/:id" />
```

Comment:
This deliberately hasn't been hooked up to any component yet, as the corresponding view hasn't been implemented